addon/utils/promise-ajax.js

```diff
@@ -0,0 +1,7 @@
+export default (options) => {
```

Comment:
If this file is only used in our own tests, can you move it to tests/helpers/?

Reply:
:+1: